roles/openshift_azure/tasks/create_image_from_vm.yml

```diff
@@ -33,8 +33,7 @@
   azure_rm_image:
     resource_group: "{{ openshift_azure_resource_group_name }}"
     source: "{{ openshift_azure_vm_name }}"
-    name: "{{ openshift_azure_image_name }}-{{ lookup('pipe', 'date +%Y%m%d%H%M%S') }}"
-
+    name: "{{ openshift_azure_image_name }}"
```

Comment:
Does this get overwritten?  How/what are you using to determine the differences between images?